src/contracts/interfaces/IRewardsCoordinator.sol

```diff
@@ -93,6 +94,18 @@ interface IRewardsCoordinator {
         uint32 duration;
     }
 
+    /// @notice Used for performance based reward submissions.
+    struct PerformanceRewardsSubmission {
+        uint32 operatorSetId;
+        address[] operators;
+        uint32[] scores;
+        IStrategy[] strategies;
```

Comment:
We may have to update this to `StrategiesAndMultipliers`. Differs from spec. Will keep you updated